app/admin/company.rb

```diff
@@ -1,5 +1,21 @@
 ActiveAdmin.register Company do
+  permit_params :name, :description, :email, :image, :address, :phone
+  menu label: 'Компании'
 
+  index do
+    selectable_column
+    id_column
+    column 'Изображение' do |company|
+      image_tag company.image_url(:small) if company.image?
+    end
+    column :email
+    column 'Название', :name
+    column 'Адрес', :address
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.